documentation/book/proc-upgrading-kafka-user-resources.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// assembly-upgrade-resources.adoc
+
+[id='proc-upgrade-kafka-user-resources-{context}']
+= Upgrading Kafka User resources
+
+.Prerequisites
+
+* The Cluster Operator is up and running.
```

Comment:
Similar comment as above, but in this case we require that the _User Operator_ is running.